tools/config-generator/templates/k8s_testgrid.yaml

```diff
@@ -0,0 +1,11 @@
+# Dashboards need to be specified here to be created on TestGrid
+# A prow annotation will be invalid if it references a dashboard that doesn't exist
+[[indent_array_section 0 "dashboards" .NamedDashboards]]
+
+dashboard_groups:
+  - name: knative
```

Comment:
If you read the names from the meta config file instead of hardcoding (potentially you will use a `map[string][]string` struct), you could use `{{range pipeline}} T1 {{end}}` to iterate over the map here.

Reply:
Didn't quite use that approach, but I think the refactor resolves this.